.golangci.yml

```diff
@@ -70,6 +70,55 @@ linters-settings:
       # Controller Runtime
       - pkg: sigs.k8s.io/controller-runtime
         alias: ctrl
+      # CABPK
+      - pkg: sigs.k8s.io/cluster-api/bootstrap/kubeadm/api/v1alpha3
+        alias: bootstrapv1alpha3
+      - pkg: sigs.k8s.io/cluster-api/bootstrap/kubeadm/api/v1alpha4
+        alias: bootstrapv1alpha4
+      - pkg: sigs.k8s.io/cluster-api/bootstrap/kubeadm/api/v1beta1
+        alias: bootstrapv1
+      # KCP
+      - pkg: sigs.k8s.io/cluster-api/controlplane/kubeadm/api/v1alpha3
+        alias: controlplanev1alpha3
+      - pkg: sigs.k8s.io/cluster-api/controlplane/kubeadm/api/v1alpha4
+        alias: controlplanev1alpha4
+      - pkg: sigs.k8s.io/cluster-api/controlplane/kubeadm/api/v1beta1
+        alias: controlplanev1
+      # CAPI
+      - pkg: sigs.k8s.io/cluster-api/api/v1alpha3
+        alias: clusterv1alpha3
+      - pkg: sigs.k8s.io/cluster-api/api/v1alpha4
+        alias: clusterv1alpha4
+      - pkg: sigs.k8s.io/cluster-api/api/v1beta1
+        alias: clusterv1
+      # CAPI exp
+      - pkg: sigs.k8s.io/cluster-api/exp/api/v1alpha3
+        alias: expv1alpha3
```

Comment:
nit: clusterexp*?

Reply:
Would be fine for me. I just used the most commonly used one. But I'm not sure if it would be better (same would probably apply to `addonsv1` => `clusteraddonsv1`, too)

@fabriziopandini WDYT?

Reply:
using cluster prefix seems kind of redundant to me, but no strong opinions

Reply:
Similar here.